docs/HowTo/Monitor/Logging.md

```diff
@@ -0,0 +1,89 @@
+---
+description: Teku log level settings and log formatting
+---
+
+# Logging
+
+Teku uses Log4J2 for logging, and provides multiple methods to configure logging behavior:
+
+* [Basic](#basic-log-level-setting) - changes the log level.
+* [Destination logging](#configure-log-destination) - configure the destination for log output.
+* [Custom logging](#advanced-custom-logging) - custom logging to configure the output and format
+    of the logs.
+
+## Basic log level setting
```

Comment:
We should have a note here that by default most logging output is sent to the log file and only limited content shown on the console.

Reply:
Done.